### PR TITLE
[BIT] 引擎修复和补齐：paddleonly 初始化 tensor 能力修复, all any 补齐

### PR DESCRIPTION
**Pull request by Cutelemon6**
### paddle.Tensor.all
\_tmp_tensor_ = args[0], result = _tmp_tensor_.all(dim=dim, keepdim=keepdim)

### paddle.Tensor.any
\_tmp_tensor_ = args[0], result = _tmp_tensor_.any(dim=dim, keepdim=keepdim)

### 引擎修复
在使用 paddle only 测试 paddle.nn.functional.max_unpool2d 时，发现引擎能力补齐的相关代码没有应用，原因是 paddle_only.py 中，初始化 tensor 的逻辑是先调用 gen_paddle_input()，该函数调用下层 get_numpy_tensor 时没有传入 index 和 key 参数，导致任务一中同学们为初始化 tensor 进行的引擎能力补齐的工作不可用（初始化 tensor 依赖调用 index 和 key 参数来识别初始化的是对应 api 的哪个 tensor）

----- update -----

为修复问题，并最小化其它代码改动，~为 gen_paddle_input() 中开始位置添加 gen_numpy_input()，这样 gen_paddle_input() 就向上层调用屏蔽了 index 和 key 的细节，并且不需要改动其它代码。利用 get_numpy_tensor 已有的实现，多次调用 get_numpy_tensor 并不会产生影响，第二次及以后调用只会返回已经初始化好的 numpy_tensor，只要调用的是同一个 TensorConfig，而 gen_paddle_input() 和 gen_numpy_input() 恰好是初始化的同一个 APIConfig 中的 TensorConfig。但是 gen_torch_input() 不确定是否和 gen_numpy_input() 初始化的是同一个 APIConfig 中的 TensorConfig，暂时未作修改，也不影响 accuracy.py 中 gen_torch_input() 的使用。~ 为 paddle_only.py 调用 gen_paddle_input() 前添加 gen_numpy_input()

```python
try:
    if not self.gen_numpy_input():
        print("gen_numpy_input failed")
        return
except Exception as err:
    print("[paddle error]", self.api_config.config, "\n", str(err))
    write_to_log("paddle_error", self.api_config.config)
    return
```

### CODE REVIEW

File: tester/base.py [hunk 1]
```diff
@@ -731,6 +736,10 @@ def _handle_list_or_tuple_torch(self, config_items, is_tuple=False):
         return tuple(tmp) if is_tuple else tmp
 
     def gen_torch_input(self):
+        # TODO: @xuyimeng
+        # check whether to add self.gen_numpy_input() in gen_torch_input()
+        # curently gen_torch_input() is only used in accuracy.py, which has gen_numpy_input() before self.gen_torch_input(). torch_args_config are only used for paddle.Tensor.__getitem__ and paddle.Tensor.__setitem__, while torch_kwargs_config are used for other apis. The main difference between gen_torch_input() and gen_paddle_input() is that gen_torch_input() init by torch_args_config instead of paddle_args_config which is used in gen_paddle_input() and gen_numpy_input().
+
```
Comment (cangtianhuang) [inline tester/base.py:742]: self.torch_args_config 等由 ana_torch_api_info 设置，由 self.paddle_args_config 等转化而来。本质上两者等价，只是部分位置参数绑定为了关键字参数

gen_numpy_input 遍历的是 self.paddle_args_config ，效果是幂等的。我建议，如果要实现屏蔽调用细节，gen_torch_input、gen_paddle_input 均应添加 gen_numpy_input，并在 accuracy 中移除单独调用的 gen_numpy_input ；否则一个添加一个不添加，其他人无法知道这三个是什么样的调用关系

Comment (cszdrg) [reply tester/base.py:742]: 那直接像accuracy一样，先gen_numpy_input 在gen_paddle_input好了 这样子逻辑最清楚 就不要套来套去了

Comment (cszdrg) [reply tester/base.py:742]: 那直接像accuracy一样，先gen_numpy_input 在gen_paddle_input好了 这样子逻辑最清楚 就不要套来套去了
不要修改base了，在paddle_only里 先gen_numpy_input一下

Comment (cangtianhuang) [reply tester/base.py:742]: 附议

Comment (Cutelemon6) [reply tester/base.py:744]: > 那直接像accuracy一样，先gen_numpy_input 在gen_paddle_input好了 这样子逻辑最清楚 就不要套来套去了 不要修改base了，在paddle_only里 先gen_numpy_input一下

已修改

**Review by wanghuancoder — APPROVED** [review]
LGTM
